apps/stack/template.yaml

```diff
@@ -378,6 +398,22 @@ Resources:
           - UseStackName
           - !Sub "${AWS::StackName}-MetricStream"
           - !Sub "${NameOverride}-MetricStream"
+  MetricsPollerRole:
+    Type: AWS::Serverless::Application
+    Condition: EnableMetricsPollerRole
+    Properties:
+      Location: ../externalrole/template.yaml
```

Comment:
You should surface the outputs of this template in the parent stack.

Reply:
Sounds good!